release/_package.sh

```diff
@@ -132,6 +132,23 @@ function package_release {
 
 	lc_cd "${_BUILD_DIR}/release"
 
+	if [[ "${_PRODUCT_VERSION}" == 7.3* ]]
+	then
+		mkdir -p liferay-portal-dependencies-"${_PRODUCT_VERSION}"
```

Comment:
@pgydev, shouldn't we name this folder as `liferay-${LIFERAY_RELEASE_PRODUCT_NAME}-dependencies-${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}"`?

Reply:
@natocesarrego  Yes, you are right we need that, but i think it's enough to put the `${_BUILD_TIMESTAMP}` to the zip file's name.